ios/HathorMobile/PrivacyInfo.xcprivacy

```diff
@@ -6,18 +6,20 @@
 	<array>
 		<dict>
 			<key>NSPrivacyAccessedAPIType</key>
-			<string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
```

Comment:
All changes to this file were done automatically by XCode. If reverted, they will be re-written on the next build.